package.json

```
@@ -41,9 +41,10 @@
     "update:deps": "ncu -u -x 'node-fetch eslint' && yarn",
     "start:server": "serve node_modules/@metamask/test-dapp/dist -p 3000",
     "test:e2e": "start-server-and-test 'yarn start:server' http-get://localhost:3000 'SKIP_METAMASK_SETUP=true SYNPRESS_LOCAL_TEST=true node synpress.js run --configFile=synpress.config.js'",
+    "test:e2e:anvil": "start-server-and-test 'yarn start:server' http-get://localhost:3000 'SKIP_METAMASK_SETUP=true SYNPRESS_LOCAL_TEST=true CYPRESS_USE_ANVIL=true node synpress.js run --configFile=synpress.config.js'",
     "test:e2e:headless": "start-server-and-test 'yarn start:server' http-get://localhost:3000 'SKIP_METAMASK_SETUP=true SYNPRESS_LOCAL_TEST=true node synpress.js run --configFile=synpress.config.js --headless'",
+    "test:e2e:headless:anvil": "start-server-and-test 'yarn start:server' http-get://localhost:3000 'SKIP_METAMASK_SETUP=true SYNPRESS_LOCAL_TEST=true CYPRESS_USE_ANVIL=true node synpress.js run --configFile=synpress.config.js --headless'",
```

This is not the way to activate headless mode, it should be like below 👇 
Unless you changed something from my latest PR!
```suggestion
    "test:e2e:headless:anvil": "start-server-and-test 'yarn start:server' http-get://localhost:3000 'HEADLESS_MODE=true SKIP_METAMASK_SETUP=true SYNPRESS_LOCAL_TEST=true CYPRESS_USE_ANVIL=true node synpress.js run --configFile=synpress.config.js'",
```

---

Aha, you updated the [code](https://github.com/Synthetixio/synpress/blob/dev/launcher.js#L35)

---

@AhmedIbrahim336 yup! mentioned it here:
https://github.com/Synthetixio/synpress/pull/654#pullrequestreview-1314586619
env variable will still work if you prefer to use it :).

---

thanks for looking in to it!